### DISCUSSION TITLE
[Assignment 2]: points

### DISCUSSION THREAD

**Discussion by alecfilios** [gstavrinos/ai_msc_robotic_systems_assignments#12] [Q&A] [answered]
How do we get the 5 + 5 points before the goals?
I get the 5 + 5 from the start and a couple of points afterward from the 4x20, but how exactly do we acquire the other 10 (5+5)?

**Comment by gstavrinos** [comment, marked as answer]
Here https://github.com/gstavrinos/ai_msc_robotic_systems_assignments/blob/97230df70cdc9096753bfd4e84b0ce2b6142d731/tt_umpire/src/tt_umpire.cpp#L221 you can see the main function of the second assignment's umpire. As you can see, the first four 5-point tasks are testing lifecycle-related stuff. So, in order to get all points, make sure you satisfy the requirements of the assignment, which are (mostly) also tested inside the umpire's code.
